dbms/tests/queries/0_stateless/01047_simple_aggregate_sizes_of_columns_bug.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS column_size_bug;
+
+CREATE TABLE column_size_bug (date_time DateTime, value SimpleAggregateFunction(sum,UInt64)) ENGINE = AggregatingMergeTree PARTITION BY toStartOfInterval(date_time, INTERVAL 1 DAY) ORDER BY (date_time);
+
+INSERT INTO column_size_bug VALUES(now(),1);
+INSERT INTO column_size_bug VALUES(now(),1);
+
+ALTER TABLE column_size_bug DELETE WHERE value=1;
+
+-- wait for DELETE
+SELECT sleep(1);
```

Comment:
That will make the test flacky (random hangs for more than one sec. is typical in our CI and using `sleep` is not viable at all). But don't worry, I'll fix it by myself.